### PR TITLE
Add selenium testing capabilities inspired from webrtc/samples

### PR DESCRIPTION
**Pull request by KaptenJansson**
* Utilizes Selenium JavaScript webdriver (and also node, tape, testling and travis multirunner) to setup and run the tests.
* Add a simple sanity test that checks that the startButton appears, can be clicked and eventually within 1 minute (might need to tweak this) of clicking on it, check that it changes state to Enabled again.
* Currently Firefox fails hence I did not enable them yet.

TODO:
Add console.log to stderr to make it easier to debug, i.e. see JS errors from the test execution..
Get firefox to pass

### CODE REVIEW

File: test/tests.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ */
+/* jshint node: true */
+/* globals require */
+
+'use strict';
+var test = require('tape');
+
+// Add all test files here with a short comment.
+
+// Tests basic functionality of the the gum demo.
```
Comment (KaptenJansson) [inline test/tests.js]: I will update this text in the next commit.

File: README.md [hunk 2]
```diff
@@ -77,12 +77,77 @@ Cleans out/ folder if it exists else it's created, then it copies and vulcanizes
 grunt build
 ```
 
-#### Run non vulcanized version of TestRTC using (Google App Engine SDK for Python)(https://cloud.google.com/appengine/downloads). This is useful while developing. ####
+#### Run non vulcanized version of TestRTC using [Google App Engine SDK for Python](https://cloud.google.com/appengine/downloads). This is useful while developing. ####
```
Comment (andresusanopinto) [inline README.md]: It starts to be too much text for the README... most of it not that relevant unless you are already quite into developing testrtc. I think we should move this to another file. Maybe more like a proper developers guide.

E.g. In the README.md I would expect a section about testing or adding new tests to be about how to add new testrtc tests and how to run only a specific test by setting test_filter=... etc... not about writing tests for the tests.

File: .travis.yml [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
 node_js:
   - 0.10
 
-install:
+env:
+  matrix:
+    - BROWSER=chrome  BVER=stable
+    - BROWSER=chrome  BVER=beta
+    - BROWSER=chrome  BVER=unstable
+    # TODO: Enable more browsers when it runs successfully.
```
Comment (andresusanopinto) [inline .travis.yml]: Create an issue to track that and remove the todo from here.

**Comment by andresusanopinto** [comment]
I am surprised by the amount of glue one needs.... didn't look in detail why all those files are needed but anyhow for now tests would be welcome.

**Comment by KaptenJansson** [comment]
>I am surprised by the amount of glue one needs.... didn't look in detail why all those files are needed but anyhow for now tests would be welcome.

Yeah it is indeed quite a lot, there are lots of components to get it to run on multiple browsers.

There is a test included here named test/sanity-test.js

File: README.md (hunk 2 above)
Comment (KaptenJansson) [inline README.md]: I originally added it as a README.md under the test folder, is that OK?

**Comment by andresusanopinto** [comment]
LGTM

**Pull request merged**